setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="jonitan-calculator",
-    version="0.1.0",
```

Comment:
Good practice Orsol

Reply:
So its good or i should remove it? i dont understand your comment Orsul

Reply:
It's good